corehq/apps/events/forms.py

```diff
@@ -249,7 +250,7 @@ def _get_possible_attendance_takers_ids(self):
             )
         else:
             users = get_all_commcare_users_by_domain(self.domain)
-        return [(u.user_id, u.username) for u in users]
+        return [(u.user_id, raw_username(u.username)) for u in users]
```

Comment:
Did you try the [`CommCareUser` property](https://github.com/dimagi/commcare-hq/blob/32579978ae848f42f0a240eda55fc0a4032eac45/corehq/apps/users/models.py#L993) for this? i.e.
```suggestion
        return [(u.user_id, u.raw_username) for u in users]
```

Reply:
@kaapstorm I did not know there's a property for this. Although technically unnecessary, I'l open another PR to change it to your suggestion, since I like it better.

Reply:
[PR here](https://github.com/dimagi/commcare-hq/pull/33042)

Reply:
Yeah, what you wrote is fine, but I agree that using the property makes more sense. Thanks for the PR!